test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -129,6 +129,13 @@ def forward(self, x):
         x = torch.randn(1, 2, 3, 4, requires_grad=True)
         self.run_test(MyModel(), x)
 
+    def test_reduce_log_sum_exp(self):
+        class ReduceLogSumExpModel(torch.nn.Module):
+            def forward(self, input):
+                return torch.logsumexp(input, dim=(0, 1))
+
+        x = torch.randn(4, 4, requires_grad=True)
+        self.run_model_test(ReduceLogSumExpModel(), x)
```

Comment:
self.run_test instead of self.run_model_test.

you can also test locally by running the command "python test/onnx/test_pytorch_onnx_onnxruntime.py TestONNXRuntime.test_reduce_log_sum_exp" :)

Reply:
D'oh, c&p fail. thanks!